packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -17,6 +17,36 @@ export const MenuItemTemplate = html<MenuItem>`
         class="${x => (x.disabled ? "disabled" : "")} ${x =>
             x.expanded ? "expanded" : ""}"
     >
+        <div part="input-container" class="input-container">
+            ${when(
+                x => x.role === MenuItemRole.menuitemcheckbox,
+                html<MenuItem>`
+                    <span part="checkbox" class="checkbox">
+                        <svg
+                            aria-hidden="true"
+                            part="checkbox-indicator"
+                            class="checkbox-indicator"
+                            viewBox="0 0 20 20"
+                            xmlns="http://www.w3.org/2000/svg"
+                        >
+                            <path
+                                fill-rule="evenodd"
+                                clip-rule="evenodd"
+                                d="M8.143 12.6697L15.235 4.5L16.8 5.90363L8.23812 15.7667L3.80005 11.2556L5.27591 9.7555L8.143 12.6697Z"
+                            />
+                        </svg>
+                    </span>
+                `
+            )}
+            ${when(
+                x => x.role === MenuItemRole.menuitemradio,
+                html<MenuItem>`
+                    <span part="radio" class="radio">
+                        <div part="radio-indicator" class="radio-indicator"></div>
```

Comment:
@khamudom one more nit on this - can you change this to a span? Technically span elements are permitted to only include phrasing content: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content

The only other question I have here is whether these should be slots w/ default slotted content so people can pass their own svg's into this. Also curious as to why we're using an SVG for the checkbox above but not here.

Reply:
I agree they should be a slot with default indicators.
I was mimicking our checkbox and radio component, just to make sure these inputs in the menu-item look similar to the stand-alone version. The radio component is using just a `div` element (which I will change to a `span` here) But, I don't see why it couldn't be an SVG. I will investigate find a "dot" SVG.

Reply:
Gotcha - can we wrap these in a slot and treat them as default slotted content for now? I think we can keep the current radio implementation, that's fine; let's just change the element to a span.

Reply:
Added default slots for the indicators and refactor/combined some selectors in the styles file. Also added some examples to show custom indicators.

![image](https://user-images.githubusercontent.com/37851220/109885516-78fdb000-7c33-11eb-8784-c90089289c3b.png)